addons/website_slides/views/slide_slide_views.xml

```diff
@@ -181,7 +209,7 @@
                     <templates>
                         <t t-name="kanban-box">
                             <div class="oe_kanban_global_click o_kanban_record_has_image_fill">
-                                <t t-set="placeholder" t-value="'/website_slides/static/src/img/channel-training-default.jpg'"/>
+                                <t t-set="placeholder" t-value="'website_slides/static/src/img/channel-training-default.jpg'"/>
```

Comment:
Why this change ?

Reply:
I was correcting placeholders for images and I modified it by accident since it is the same image that is used as the placeholder for new content. Rectifying it.